Sources/StreamCore/Query/Filter+Local.swift

```diff
@@ -155,27 +148,29 @@ private struct FilterMatcher<Model, Value>: Sendable where Model: Sendable, Valu
     static func query(_ localRawJSONValue: RawJSON, _ filterRawJSONValue: RawJSON) -> Bool {
         switch (localRawJSONValue, filterRawJSONValue) {
         case (.string(let localStringValue), .string(let filterStringValue)):
-            return localStringValue.localizedCaseInsensitiveContains(filterStringValue)
+            return Self.postgreSQLFullTextSearch(anchored: false, text: localStringValue, query: filterStringValue)
```

Comment:
If I read this correctly, this is equivalent to checking that `filterStringValue` is not empty && that it is contained in `localStringValue` (case insensitive). If contains on the whole string fails, then it will fail in substrings too, right?